src/containerapp/azext_containerapp/_params.py

```diff
@@ -113,6 +113,10 @@ def load_arguments(self, _):
         c.argument('enable_dedicated_gpu', arg_type=get_three_state_flag(), options_list=["--enable-dedicated-gpu"],
                    help="Boolean indicating if the environment is enabled to have dedicated gpu", is_preview=True)
 
+    with self.argument_context('containerapp env create', arg_group='Identity', is_preview=True) as c:
+        c.argument('system_assigned', options_list=['--mi-system-assigned'], help='Boolean indicating whether to assign system-assigned identity.', action='store_true')
```

Comment:
>--mi-system-assigned [](http://example.com/codeflow?start=53&length=20)

use --system-assigned, which is consistent with container app one

Reply:
discussed offline